src/main/java/com/github/cameltooling/lsp/internal/instancemodel/PathParamURIInstance.java

```diff
@@ -86,4 +87,14 @@ private String getFilter() {
 		}
 		return filter;
 	}
+	
+	@Override
+	public String getComponentName() {
+		return uriInstance.getComponentName();
+	}
+	
+	@Override
+	public String getDescription(ComponentModel componentModel) {
+		return componentModel.getEndpointOption(value).getDescription();
```

Comment:
I think it is here that you need to check for non-nullity to avoid NPE when an unknown attribute is used